src/utils/gp_lstm.py

```diff
@@ -0,0 +1,47 @@
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+
+
+class gaussian_lstm(nn.Module):
```

Comment:
Can you explain this? Do we need the embedding layer? 
I think the LSTM and the gaussian process can be separate